index.html

```diff
@@ -34,6 +34,35 @@
         <a href="#about"><p>About</p></a>
         <a href="#contact"><p>Contact</p></a>
       </nav>
+    </div>
+    <div id="overlay">s
+      <div id="popup">
+        <div class="popup-header">
```

Comment:
- [ ] Please set more space on the top and bottom of header text, increase the font size as provided on the design: 

Your design  | Figma design
------------- | -------------
<img width="340" alt="Screenshot 2022-09-09 at 15 41 31" src="https://user-images.githubusercontent.com/77038610/189352246-872d22f9-fd96-40aa-b8d4-42d25c974263.png"> | <img width="262" alt="Screenshot 2022-09-09 at 15 42 02" src="https://user-images.githubusercontent.com/77038610/189352327-e3230a3e-77b6-4917-99f1-712c62777036.png">

Reply:
The project title is aligned with the image, spaces increase with font size.